plugin/ensime.vim

```diff
@@ -12,6 +12,7 @@ augroup END
 
 command! -nargs=* -range EnNoTeardown call ensime#com_en_no_teardown([<f-args>], '')
 command! -nargs=* -range EnTypeCheck call ensime#com_en_type_check([<f-args>], '')
+command! -nargs=* -range EnBufferedTypeCheck call ensime#com_en_buf_type_check([<f-args>], '')
```

Comment:
Good point, I would tend to agree. It could start as an option that you can toggle and then become the only supported `typecheck`.